src/main/assembly/dist/md/emd/2017/09/eas.xsd

```diff
@@ -0,0 +1,353 @@
+            <xs:element name="subject-link" type="xs:anyURI" minOccurs="0"/>
+        </xs:sequence>
+        <xs:attribute name="emphasis" type="xs:boolean" default="false"/>
+    </xs:complexType>
+
+    <xs:complexType name="authortype">
+        <xs:sequence>
+            <xs:element name="title" type="xs:string" minOccurs="0"/>
+            <xs:element name="initials" type="xs:string" minOccurs="0"/>
+            <xs:element name="prefix" type="xs:string" minOccurs="0"/>
+            <xs:element name="surname" type="xs:string" minOccurs="0"/>
+            <xs:element name="organization" type="xs:string" minOccurs="0"/>
+            <xs:element name="entityId" minOccurs="0">
+                <xs:complexType>
+                    <xs:simpleContent>
+                        <xs:extension base="xs:string">
+                            <xs:attribute ref="eas:identification-system"/>
+                            <xs:attribute name="scheme" type="xs:string" default="DAI"/>
+                        </xs:extension>
+                    </xs:simpleContent>
+                </xs:complexType>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:element name="point">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="x" type="xs:string" minOccurs="0"/>
+                <xs:element name="y" type="xs:string" minOccurs="0"/>
+            </xs:sequence>
+            <xs:attribute name="scheme" type="xs:token"/>
+            <xs:attribute name="schemeId" type="xs:string"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="box">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="north" type="xs:string" minOccurs="0"/>
+                <xs:element name="east" type="xs:string" minOccurs="0"/>
+                <xs:element name="south" type="xs:string" minOccurs="0"/>
+                <xs:element name="west" type="xs:string" minOccurs="0"/>
+            </xs:sequence>
+            <xs:attribute name="scheme" type="xs:token"/>
+            <xs:attribute name="schemeId" type="xs:string"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="polygon">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="eas:polygon-exterior" minOccurs="0"/>
+                <xs:element ref="eas:polygon-interior" minOccurs="0" maxOccurs="unbounded"/>
+            </xs:sequence>
+            <xs:attribute name="scheme" type="xs:token"/>
+            <xs:attribute name="schemeId" type="xs:string"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="polygon-exterior" type="eas:PolygonPointType"/>
+    <xs:element name="polygon-interior" type="eas:PolygonPointType"/>
+
+    <xs:complexType name="PolygonPointType">
+        <xs:sequence>
+            <xs:element name="place" type="xs:string" minOccurs="0"/>
```

Comment:
there is already a 'place' in the 'spatial type' (also note the lowercase 't' in "type").

Reply:
I know. However, the GML polygon defines `place` (called `description` there) on a `PolygonPartType` as well as on the whole `Polygon`.